1129/en.md

```diff
@@ -0,0 +1,107 @@
+# 1129 - Consistency Checker
```

Comment:
Please follow the format we use for the problem heading - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR1